src/lib.rs

```diff
@@ -400,6 +433,25 @@ impl Tag {
                     inner.add_picture(&pic)?;
                 }
             }
+            Self::OggTag { inner } => {
+                if let Some(title) = album.title {
+                    inner.comments.insert("album".into(), vec![title]);
+                }
+                if let Some(album_artist) = album.artist {
+                    inner
+                        .comments
+                        .insert("album_artist".into(), vec![album_artist]);
+                }
+                if let Some(picture) = album.cover {
+                    // Try to decode the image to obtain width/height and color depth
+                    let dyn_img = image::load_from_memory(picture.data.as_slice())
```

Comment:
Do we need to do the image loading ourselves? This seems like something oggmeta should handle.